docs/docs/api/http.md

```diff
@@ -643,3 +643,24 @@ The response comes in two parts:
   - `total`
 
     The total number of elements across all pages.
+
+### Metadata
+
+Please refer to our [metadata](glossary.md#metadata) definition for more
+information.
+
+### Setting metadata
+
+Airy provides a mechanism to add metadata to a conversation.
```

Comment:
```suggestion
```

Yes I'm suggesting to remove... but mostly because I cant come up with anything constructive and probably we can get away without it anyway (if we want to keep a description... we'll have to change the subject because it reads as we as a company provide that but it's the platform)